gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -573,7 +550,7 @@ public void modifyAckDeadline(ModifyAckDeadlineRequest request) {
    * @param ackIds The acknowledgment ID for the messages being acknowledged that was returned
    * by the Pub/Sub system in the `Pull` response. Must not be empty.
    */
-  public void acknowledge(String subscription, List<String> ackIds) {
+  public final void acknowledge(String subscription, List<String> ackIds) {
```

Comment:
Can you specify the reason why we should choose Iterable over List here? Currently we always use List for paramenters across the files so it would be good to keep the consistency. And I do feel List sometimes is more flexible to use than Iterable.